src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -572,6 +731,142 @@ void CoilCoolingDX::simulate(EnergyPlus::EnergyPlusData &state, int useAlternate
+            dummyEvapInlet.Press = DataEnvironment::StdPressureSeaLevel;
+            dummyEvapInlet.HumRat = dummyInletAirHumRat;
+            dummyEvapInlet.Enthalpy = Psychrometrics::PsyHFnTdbW(RatedInletAirTemp, dummyInletAirHumRat);
+
+            // maybe we don't actually need to override weather below, we'll see
+            dummyCondInlet.Temp = RatedOutdoorAirTemp;
+            dummyCondInlet.HumRat =
+                Psychrometrics::PsyWFnTdbTwbPb(state, RatedOutdoorAirTemp, ratedOutdoorAirWetBulb, DataEnvironment::StdPressureSeaLevel, RoutineName);
+            dummyCondInlet.OutAirWetBulb = ratedOutdoorAirWetBulb;
+            dummyCondInlet.Press = condInletNode.Press; // for now; TODO: Investigate
+
+            // overriding outdoor conditions temporarily
+            Real64 holdOutDryBulbTemp = state.dataEnvrn->OutDryBulbTemp;
+            Real64 holdOutHumRat = state.dataEnvrn->OutHumRat;
+            Real64 holdOutWetBulb = state.dataEnvrn->OutWetBulbTemp;
+            Real64 holdOutBaroPress = state.dataEnvrn->OutBaroPress;
+            state.dataEnvrn->OutDryBulbTemp = RatedOutdoorAirTemp;
+            state.dataEnvrn->OutWetBulbTemp = ratedOutdoorAirWetBulb;
+            state.dataEnvrn->OutBaroPress = DataEnvironment::StdPressureSeaLevel; // assume rating is for sea level.
+            state.dataEnvrn->OutHumRat =
+                Psychrometrics::PsyWFnTdbTwbPb(state, RatedOutdoorAirTemp, ratedOutdoorAirWetBulb, DataEnvironment::StdPressureSeaLevel, RoutineName);
+
+            this->performance.simulate(state,
+                                       dummyEvapInlet,
+                                       dummyEvapOutlet,
+                                       false,
+                                       dummyPLR,
+                                       dummySpeedNum,
+                                       dummySpeedRatio,
+                                       dummyFanOpMode,
+                                       dummyCondInlet,
+                                       dummyCondOutlet,
+                                       dummySingleMode);
+
+            // reset outdoor conditions back to previous state
+            state.dataEnvrn->OutDryBulbTemp = holdOutDryBulbTemp;
+            state.dataEnvrn->OutWetBulbTemp = holdOutWetBulb;
+            state.dataEnvrn->OutBaroPress = holdOutBaroPress;
+            state.dataEnvrn->OutHumRat = holdOutHumRat;
+
+            Real64 const coolingRate = dummyEvapInlet.MassFlowRate * (dummyEvapInlet.Enthalpy - dummyEvapOutlet.Enthalpy);
+            Real64 const thisMinAirHumRat = min(dummyEvapInlet.HumRat, dummyEvapOutlet.HumRat);
+            Real64 const sensCoolingRate = dummyEvapInlet.MassFlowRate * (Psychrometrics::PsyHFnTdbW(dummyEvapInlet.Temp, thisMinAirHumRat) -
+                                                                          Psychrometrics::PsyHFnTdbW(dummyEvapOutlet.Temp, thisMinAirHumRat));
```

Comment:
Didn't we have a different calculation for sens cap? A new @Nigusse function somewhere?